web-app/js/portal/cart/DownloadPanelItemTemplate.js

```diff
@@ -41,6 +47,36 @@ Portal.cart.DownloadPanelItemTemplate = Ext.extend(Ext.XTemplate, {
         ];
     },
 
+    _getHtmlErrorContent: function() {
+        return [
+            '<div class="downloadPanelResultsWrapper">',
+            '  <div class="x-panel-header downloadPanelResultsTitle">',
+            '    <div class="downloads resultsRowHeaderTitle">',
+            '      <div class="x-tool-awesome" title="{[this.getTooltip(\'removeDataCollectionTooltip\')]}" class="removeButton" id="{[this._getLinkId(values,\'removeButtonId\')]}">{[this._createRemoveButtonAfterPageLoad(values)]}</div>',
+            '      <div class="x-tool-awesome" title="{[this.getTooltip(\'shareButtonTooltip\')]}" id="{[this._getLinkId(values,\'shareButtonId\')]}">{[this._createShareButtonAfterPageLoad(values)]}</div>',
+            '    <span class="x-panel-header-text">{[this._getHtmlTitle(values)]}</span>',
+            '    </div>',
+            '  </div>',
+            '  <div style="overflow:hidden;">',
+            '    <div class="message">'+OpenLayers.i18n('emptyMsg')+'</div>',
```

Comment:
Did you consider conditionally including this element using tpl if instead of duplicating the whole template?  (refer https://docs.sencha.com/extjs/3.4.0/#!/api/Ext.XTemplate)